src/main/java/com/elixr/poc/rest/controller/PurchaseDeletionController.java

```diff
@@ -38,11 +38,11 @@ public ResponseEntity<?> deletePurchase(@PathVariable("purchaseId") UUID purchas
         try {
             boolean success = purchaseService.deletePurchaseDetails(purchaseId);
             deleteResponse.setSuccess(success);
-            deleteResponse.setErrorMessages(ApplicationConstants.DELETED);
+            deleteResponse.setErrorMessage(ApplicationConstants.SUCCESSFULLY_DELETED);
             httpStatus = HttpStatus.OK;
         } catch (NoRecordFoundException e) {
```

Comment:
Follow the same pattern as in DeletionController for handling NoRecordFoundException.
This method just throws that exception. Its handled in GLobalExeptionHandler

Reply:
Updated.